CorsixTH/CMakeLists.txt

```diff
@@ -14,18 +14,23 @@ if(MSVC)
   add_definitions(-D_CRT_SECURE_NO_WARNINGS)
 endif()
 
+if(SEARCH_LOCAL_DATADIRS)
+  add_definitions(/DCORSIX_SEARCH_LOCAL_DATADIRS)
```

Comment:
`-D` for consistency, or better yet:
`set(CORSIX_TH_SEARCH_LOCAL_DATADIRS ${SEARCH_LOCAL_DATADIRS)` here
`#cmakedefine CORSIX_TH_SEARCH_LOCAL_DATADIRS` in config.h.in

Note `CORSIX_TH_` not `CORSIX_` as the prefix, matching all other defines.